test/ngMock/angular-mocksSpec.js

```diff
@@ -2505,6 +2505,44 @@ describe('`afterEach` clean-up', function() {
     });
   });
 
+  describe('ErrorAddingDeclarationLocationStack', function() {
+    it('should be caught by jasmine\'s toThrowError', function() {
+      //create function that causes an error
+      function causeNgMocksError() {
+        module(function($provide) {
+          //cause $provide.factory to include an error
+          $provide.factory('causeError', function() {
+            //Say for example, some mock module caused an error when instantiated
+            throw new Error("Error!");
+          });
+        });
+
+        //call the injector on 'causeError' to make sure the provider is called
+        var causeError;
+        inject(function(_causeError_) {
+          causeError = _causeError_;
+        });
+      }
+
+      //The following is a simplified implementation of future Jasmine's .toThrowError()
```

Comment:
What do you mean by `future Jasmine's .toThrowError()` ?
Why can't you use it directly ?